### ISSUE TITLE
Error: card not found

### ISSUE THREAD

**Issue by nickserv** [PixR2/mtg-code#1]
I'm trying this extension on a standard Kamigawa deck, but I get an error on each card:
> failed to get card from card database: Error: card not found

**Comment by PixR2** [comment]
Thank you for opening a ticket. I will look at the issue tomorrow. Most likely the scryfall API changed slightly, which causes an error while parsing responses. I wanted to make the parsing less strict bit have not had time for it. Right now it fails on any unexpected value or type, which breaks everything when scryfall changes just small details.

**Comment by PixR2** [comment]
Okay I played a bit with some Neon Dynasty lists I found online. I encountered a problem with double sided cards, which in most deck lists are only given by the name of the front face. Right now the card lookup in mtg-code expects the exact card name as stored in scryfalls DB. As an example the line:
`3 Emeria's Call`
will result in the error you are seeing. But the line:
`3 Emeria's Call // Emeria, Shattered Skyclave`
works.
Is this the problem you are seeing?
